ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -271,6 +280,18 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz-sts
+- as: aws-ipi-shared-vpc-phz-sts-openldap-fips-f28
```

Comment:
I think most configuration of this job is the same as `aws-ipi-shared-vpc-phz-sts-fips-f14`, I think we need to drop `aws-ipi-shared-vpc-phz-sts-fips-f14` if we are using the new one to run upgrade testing.